src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -103,6 +103,11 @@ @implementation GRPCCall {
 
 @synthesize state = _state;
 
+// TODO(jcanizales): If grpc_init is idempotent, this should be changed from load to initialize.
++ (void)load {
```

Comment:
Not needed! :) `load` and `initialize` [are magical](http://stackoverflow.com/questions/13326435/nsobject-load-and-initialize-what-do-they-do).